app/src/test/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentServiceIT.java

```diff
@@ -80,6 +81,7 @@ void cleanDatabaseTables() {
                 new ClassPathResource("fixtures/scxa_cell_group_membership-delete.sql"),
                 new ClassPathResource("fixtures/scxa_cell_group-delete.sql"),
                 new ClassPathResource("fixtures/scxa_coords-delete.sql"),
+                new ClassPathResource("fixtures/scxa_dimension_reduction-delete.sql"),
```

Comment:
Here `scxa_dimension_reduction` comes first 
         
```
new ClassPathResource("fixtures/scxa_dimension_reduction-delete.sql"),
new ClassPathResource("fixtures/scxa_coords-delete.sql"),..
```

Reply:
Please ignore if the cleanup script wasn't throwing any errors.

Reply:
Deletions should be the inverse order of insertions.